src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -30,20 +32,27 @@ public void setUp() {
     @Test
     public void execute_newPerson_success() {
         Person validPerson = new PersonBuilder().build();
-
-        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
-        expectedModel.addPerson(validPerson);
-        expectedModel.commitAddressBook();
-
-        assertCommandSuccess(new AddCommand(validPerson), model, commandHistory,
-                String.format(AddCommand.MESSAGE_SUCCESS, validPerson), expectedModel);
+        try {
+            Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+            expectedModel.addPerson(validPerson);
+            expectedModel.commitAddressBook();
+
+            assertCommandSuccess(new AddCommand(validPerson), model, commandHistory,
+                    String.format(AddCommand.MESSAGE_SUCCESS, validPerson), expectedModel);
+        } catch (NoUserSelectedException e) {
```

Comment:
This should not be here because this will never happen in the test case. If it happens, something is wrong.